File: Database_PHP/api/Sport.php
<?php
    class Sport{

        public function __construct($con)
        {
            $this->con=$con;
        }

        public function insertSport($name,$image, $technique, $benefit, $note){

            $result=mysqli_query($this->con, "INSERT INTO sports VALUES ('','$name','$image','$technique','$benefit','$note')");
            return $result;
        }

        public function updateSport($id, $name,$image, $technique, $benefit, $note){
            $query = "UPDATE sports SET name='$name', image='$image', technique='$technique', benefit='$benefit', note='$note' WHERE id='$id'";
            $result=mysqli_query($this->con, $query);
            return $result;
        }

        public function deleteSport($id){
            $query = "DELETE FROM sports WHERE id='$id'";
            $result=mysqli_query($this->con, $query);
            return $result;
        }
    }
?>

File: lib/models/sport.dart

class SportModel{
  final String id;
  final String name;
  final String image;
  final String technique;
  final String benefit;
  final String note;

  SportModel({this.id, this.name, this.image, this.technique, this.benefit, this.note});

  factory SportModel.fromJson(Map<String, dynamic> json) {
    return SportModel(
        id: json['id'],
        name: json['name'].toString(),
        image: json['image'].toString(),
        technique: json['technique'].toString(),
        benefit: json['benefit'].toString(),
        note: json['note'].toString()
    );
  }
}